lib/crutches/format/number.ex

```diff
@@ -102,7 +102,7 @@ defmodule Crutches.Format.Number do
   # Examples
 
       iex> Number.as_rounded(111.2345)
-      "111.235"
+      "111.234"
```

Comment:
There have been changes to `Float.ceil/2`, `Float.floor/2` and `Float.round/2` in Elixir 1.4. The test-case covered the old behaviour. I'd rather not break backwards compatibility here: the old result is probably what users would expect (but any input regarding this is welcome!)

I think it is best if we remove this change and consider this as a bug on 1.4